modules/Neo4jDataAccess.py

```diff
@@ -521,7 +522,7 @@ def save_twintdf_to_neo(self, df, job_name, job_id=None):
                                              'user_friends_count': row[
                                                  'user_friends_count'] if 'user_friends_count' in row else None,
                                              'user_created_at': pd.to_datetime(
-                                                 df['user_created_at']) if 'user_created_at' in row else None,
+                                                 row['join_datetime']) if 'join_datetime' in row else None,
                                              'user_profile_image_url': row[
```

Comment:
join_datetime seems to be in https://github.com/twintproject/twint/blob/master/twint/storage/panda.py#L125 , but the rest of the fields here aren't... surprised this works?  but guessing manual tests were fine, so will just note it here in case need to revisit

Reply:
i removed it this morning , forgot i did account enrichments at a diff stage in the transformation when i committed this.

Reply:
going to do a round trip into neo today on everything